xla/xla_data.proto

```diff
@@ -931,7 +931,10 @@ message PrecisionConfig {
     // Each U8/S8 value in a tensor actually represents 2 nibble values.
     PACKED_NIBBLE = 3;
 
-    // Next: 4
+    ACTIVATION = 4;
```

Comment:
Please add a comment to explain the new enum values. Are ACTIVATION and GRADIENT best added here? It seems orthogonal to precision scale from DEFAULT to HIGHEST in this enum.

Reply:
Currently PrecisionConfig contains one field containing precision information for each input. This adds another field containing information whether this is a gradient or an activation. This is perhaps not the most natural place to keep this, but any alternative would require passing precision in one object and gradient flags in another object, which is precisely what I am trying to avoid.

Reply:
Please document the intended usage of this enum. Why are you trying to avoid adding a gradient flag?